src/components/Navbar.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable react/prop-types */
```

Comment:
- [ ]  Kindly correct your linters instead of disabling the. they help you prevent future bugs in the app and help you apply best practices.